src/app/(auth)/login/loginModal.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+import {
+  AuthButton,
+  AuthModalContainer,
+  AuthModalContent,
+  HeadingTypography,
+  ParaTypography,
+  StyledBackdrop,
+  StyledLinkOne,
+  StyledLinkTwo,
+  StyledTextField,
+} from '../authModel.style';
+
+interface LoginModalProps {
+  open: boolean;
+  handleClose: () => void;
+}
+
+const LoginModal: React.FC<LoginModalProps> = ({ open, handleClose }) => {
+  return (
+    <AuthModalContainer
+      open={open}
+      onClose={handleClose}
+      BackdropComponent={StyledBackdrop}
+      disableScrollLock={false}
+    >
+      <AuthModalContent>
+        <HeadingTypography variant='h6'>Login</HeadingTypography>
+        <StyledTextField
+          label='Email'
+          variant='standard'
+          fullWidth
+          margin='normal'
+        />
+        <StyledTextField
+          label='Password'
+          type='password'
```

Comment:
Please make and integrate the zod validation schemas for these and use react-hook-forms for all the forms

Reply:
Additionally, when wriitng the handleSubmit functionality, use the MUI snackbar for success/error notifications where async functionalities are concerned.